WordPressKitTests/RemotePostTests.swift

```diff
@@ -0,0 +1,30 @@
+import XCTest
+import WordPressKit
+
+class RemotePostTests: RemoteTestCase, RESTTestable {
+
+    private var remote: PostServiceRemoteREST!
+
+    override func setUp() {
+        remote = PostServiceRemoteREST(wordPressComRestApi: getRestApi(), siteID: 123)
+    }
```

Comment:
Should we be calling `super` here?

```suggestion
    override func setUp() {
        super.setUp()
        remote = PostServiceRemoteREST(wordPressComRestApi: getRestApi(), siteID: 123)
    }
```

Might not be needed with the current test but could save debug time in the future.

https://github.com/wordpress-mobile/WordPressKit-iOS/blob/7d28260494b13f66e7addec518a6bd1d90867d69/WordPressKitTests/RemoteTestCase.swift#L27-L30

Reply:
Good catch! Thanks!